pkg/template/generate.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+set -e
```

Comment:
Should we move this script to a invoke task?

Reply:
Also, should we take into consideration Windows as well in the script?

Reply:
Yeah I'll rewrite this as an invoke task, will be cleaner than a shell script, and more portable too